colossalai/shardformer/modeling/bloom.py

```diff
@@ -719,7 +719,7 @@ def forward(
     ):
         fused_qkv = self.query_key_value(hidden_states)
         (query_layer, key_layer, value_layer) = self._split_heads(fused_qkv)
-        batch_size, tgt_len, _ = query_layer.size()
+        batch_size, tgt_len, _, _ = query_layer.size()
```

Comment:
Why does the shape change?

Reply:
There is a bug here, not sure why it wasn't detected before.